stl/inc/format

```diff
@@ -1116,6 +1116,47 @@ _OutputIt _Write(_OutputIt _Out, monostate) {
     return _Out;
 }
 
+// This size is derived from the maximum length of an arithmetic type. The two contenders for widest are double and long
+// long. long long has a max length of ceil(log_10(2^64)) = 20 characters. double has a max length of
+// limits<double>::max_digits10 + the decimal + the sign + e + the exponent's sign + ceil(log_10(DBL_MAX_10_EXP))
+// = 17 + 1 + 1 + 1 + 3 = 24. An example is DBL_MAX which is "1.7976931348623158e+308".
+inline constexpr size_t _Format_min_buffer_length = 24;
+
+// clang-format off
+template <class _CharT, class _OutputIt, class _Arithmetic>
+    requires (is_arithmetic_v<_Arithmetic> && !same_as<_Arithmetic, bool> && !same_as<_Arithmetic, _CharT>)
+_OutputIt _Write(_OutputIt _Out, _Arithmetic _Value) {
+    // clang-format on
+    // TODO: Reusable buffer

```

Comment:
Nitpick: Conventionally we use `TRANSITION,`

Reply:
Sorry on which part? You commented later also, I assume on the `TODO`? So `TODO: TRANSITION, Reusable buffer`?

Reply:
No it is `// TRANSITION, Reusable buffer`

Reply:
We avoid `TODO` and `FIXME` so things in the STL headers don't show up in the handy list of tasks that VS generates for customers by scraping code for occurrences of `TODO` and `FIXME`. (Well, we avoid `FIXME` in general because it suggests the code isn't ready to release.)

Reply:
There were a couple of `TODO`s in the file, I'll fix these ones.